app/javascript/components/server-components/CheckoutPage.tsx

```diff
@@ -673,27 +674,31 @@ export const CrossSellModal = ({
     <>
       <div className="grid gap-4">
         <h4 dangerouslySetInnerHTML={{ __html: crossSell.description }} />
-        <article className="product-card horizontal">
-          <figure>{product.thumbnail_url ? <img src={product.thumbnail_url} /> : null}</figure>
-          <section>
-            <header>
+        <ProductCard className="lg:grid-cols-[auto_1fr] lg:grid-rows-none">
+          <ProductCardFigure className="lg:rounded-l lg:rounded-tr-none lg:border-r lg:border-b-0">
+            {product.thumbnail_url ? (
+              <img src={product.thumbnail_url} className="h-full w-full object-cover lg:h-0 lg:min-h-full" />
+            ) : null}
+          </ProductCardFigure>
+          <ProductCardSection>
+            <ProductCardHeader className="lg:grid-rows-[repeat(auto-fit,minmax(0,min-content))] lg:border-b-0 lg:p-0">
```

Comment:
These grid rows seem unnecessarily complicated for two children, is there something simpler we can do? (e.g. use flex)